templates/content.mustache

```diff
@@ -9,15 +9,9 @@
     		{{> title.mustache}}
     	{{/unless}}
 
-      <!--
-      <ul class="on-this-page fixed open">
-        <li><a href="#">Monkeys</a></li>
-        <li class="active"><a href="#">Fish</a></li>
-        <li><a href="#">Cats</a></li>
-        <li><a href="#">Dogs</a></li>
+      <ul class="on-this-page">
```

Comment:
Even if it can't be seen on the page, an empty container is wasteful. But depending on any CSS which applies to the container in general, it could easily have margins, padding, even a border. ULs in particular usually have default padding and margins. I don't know why devs seem to avoid putting conditionals around stuff like this, but I have seen it cause problems before. It's a best practice, imo, to not print HTML that isn't needed